forecast-admin/forecast/opportunities/admin.py

```diff
@@ -1,12 +1,26 @@
 from django.contrib import admin
 
 # Register your models here.
-from .models import Opportunity, Office
+from .models import Opportunity, Office, OSBUAdvisor
 
 
 @admin.register(Opportunity)
 class OpportunityAdmin(admin.ModelAdmin):
 
+    def make_published(self, request, queryset):
```

Comment:
@khandelwal Any idea how to write unit tests for this? I've spent some time in the django docs and in stackoverflow but am coming up broke for actual answers.